embark/users/test_users.py

```diff
@@ -0,0 +1,46 @@
+from django.test import TestCase
+from django.test import Client
+from users.models import User
+
+
+class TestUsers(TestCase):
+    def setUp(self) -> None:
+        user = User.objects.create(username='testuser')
+        user.set_password('12345')
+        user.save()
+        self.client = Client()
+
+    def test_signup(self):
+        pass
+
+    def test_signin(self):
+        """
+        Right password would redirect to home
+        Returns:
+
+        """
+        response = self.client.post('/signin', {'email': 'testuser', 'password': '12345'})
+        self.assertEqual(response.status_code, 302)
+
+    def test_signin_wrong_password(self):
+        """
+        Wrong password would render the same page again with 200 status code
+        Returns:
+
+        """
+        response = self.client.post('/signin', {'email': 'testuser', 'password': '1234'})
+        self.assertEqual(response.status_code, 200)
```

Comment:
You could use http.HTTPStatus.OK for readability from https://docs.python.org/3/library/http.html#module-http.

Reply:
Done